frontend/components/TooltipWrapperArchLinuxRolling/TooltipWrapperArchLinuxRolling.tsx

```diff
@@ -3,7 +3,13 @@ import React from "react";
 import TooltipWrapper from "../TooltipWrapper";
 import CustomLink from "../CustomLink";
 
-export default function TooltipWrapperArchLinuxRolling() {
+type Props = {
```

Comment:
Nit for future - we usually use `interface`s for component props

Reply:
[Use interface instead of type](https://github.com/fleetdm/fleet/pull/33315/commits/d34790a188d48d928c7bff05ea6c0dd0d116ad25)